tests/test_commands.py

```diff
@@ -43,8 +43,9 @@ def custom(var):
 
 
 def test_custom_command_with_int_arguments():
-    def add(a: int, b: int):
+    def add(a: int, b: int) -> int:
         click.echo(str(a + b))
+        return (a+b)
```

Comment:
We don't need to return a value here. I reckon just use `-> None` and don't return anything.